### PR TITLE
RTEMIS-55: Added the user info & school info on profile view.

### PR DESCRIPTION
**Pull request by kunal-sin**
Ticket # : https://innoraft.atlassian.net/browse/RTEMIS-55

---
### What the PR does
`(Explain fixes, changes and impacts)`
- Added the user info & school info on profile view.

---
### Extras
`(Add Screenshots or videos)`

screenshot-rte-mis lndo site-2024 03 10-23_38_09

### CODE REVIEW

File: config/install/core.entity_view_display.user.user.full.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+              label_display: visible
+              provider: views
+              context_mapping: {  }
+              views_label: ''
+              items_per_page: none
+            weight: 0
+            additional: {  }
+        third_party_settings: {  }
+_core:
+  default_config_hash: U3-Q2lXuo4Q4uWyZOzcOPQyNmDWes2sBz50AAY_0A-0
```
Comment (jaswinsingh) [inline config/install/core.entity_view_display.user.user.full.yml:59]: Can we remove this?

File: config/install/core.entity_view_display.user.user.full.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    allow_custom: false
+    sections:
+      -
+        layout_id: layout_onecol
+        layout_settings:
+          label: ''
+          context_mapping: {  }
+        components:
+          a6e915f0-ecf4-4f27-a9ac-509772a9875b:
+            uuid: a6e915f0-ecf4-4f27-a9ac-509772a9875b
```
Comment (jaswinsingh) [inline config/install/core.entity_view_display.user.user.full.yml:32]: Not sure above this uuid, if this is required

File: config/install/core.entity_view_display.user.user.full.yml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+              label: ''
+              label_display: visible
+              provider: views
+              context_mapping: {  }
+              views_label: ''
+              items_per_page: none
+            weight: 1
+            additional: {  }
+          392f6897-e6e1-474f-8d9f-55ec1a2a49c4:
+            uuid: 392f6897-e6e1-474f-8d9f-55ec1a2a49c4
```
[the remark this answers is not in the document]
Comment (kunal-sin) [reply config/install/core.entity_view_display.user.user.full.yml:45]: Yes, required.

File: config/install/core.entity_view_display.user.user.full.yml (hunk 2 above)
Comment (kunal-sin) [reply config/install/core.entity_view_display.user.user.full.yml:32]: Yes, required.

File: config/install/core.entity_view_display.user.user.full.yml (hunk 1 above)
Comment (kunal-sin) [reply config/install/core.entity_view_display.user.user.full.yml]: Updated!!

**Pull request merged**